sdk/src/androidTest/java/ly/count/android/sdk/UtilsInternalLimitsTests.java

```diff
@@ -503,4 +503,27 @@ public void applySdkInternalLimitsToSegmentation_clashingKeys() {
         Assert.assertEquals(1, segmentation.get("a"));
         Assert.assertEquals("dd", segmentation.get("bb"));
     }
+
+    @Test
+    public void applyInternalLimitsToStackTraces() {
+        String stackTrace = "java.lang.Exception: test\n"
+            + "at ly.count.android.sdk.UtilsInternalLimitsTests.applyInternalLimitsToStackTraces(UtilsInternalLimitsTests.java:1)\n"
+            + "at ly.count.android.sdk.UtilsInternalLimitsTests.applyInternalLimitsToStackTraces(UtilsInternalLimitsTests.java:2)\n"
+            + "at ly.count.android.sdk.UtilsInternalLimitsTests.applyInternalLimitsToStackTraces(UtilsInternalLimitsTests.java:3)\n";
+
+        String truncatedStackTrace = UtilsInternalLimits.applyInternalLimitsToStackTraces(stackTrace, 2, "tag", new ModuleLog());
+        Assert.assertEquals("ja\nat\nat\nat", truncatedStackTrace);
```

Comment:
as in here. is there a current example how we send the stacktrace? and how does it look at the server with and without new line for the last line?

Reply:
no let me prep

Reply:
![image](https://github.com/Countly/countly-sdk-android/assets/57103426/355bc9a4-a0a8-4d53-9eff-e26c950e100a)
without new line at the end of the last line

Reply:
its cool then